doc/guide/admin.rst

```diff
@@ -681,18 +681,48 @@ complete but before resources are returned to the scheduler.
 The configuration of a prolog and epilog that runs as root requires the
 following steps:
 
- 1. Configure the IMP such that it will allow the system instance user
-    to execute a prolog and epilog script or command as root.
+ 1. Create executable scripts named ``prolog`` and ``epilog`` in
+    ``/etc/flux/system``.  A suggested approach is to have these scripts run
+    all executables found in ``prolog.d`` and ``epilog.d``.  For example:
+
+    .. code-block:: sh
+
+       #!/bin/sh
+
+       exit_rc=0
+       periname=prolog
+       peridir=/usr/flux/system/${periname}.d
+
+       test $FLUX_JOB_USERID && userid=$(id -n -u $FLUX_JOB_USERID 2>/dev/null)
```

Comment:
It might be more common to just exit with an error if `FLUX_JOB_USERID` or `FLUX_JOB_ID` are not set, since this indicates the prolog/epilog is not running under Flux. Then the rest of the script could assume these variables are set.

Reply:
I had that in because you can start the prolog and epilog manually for testing, e.g.
```
$ sudo flux run-epilog
```
and then the journal captures
```
$ journalctl -u flux-epilog@*
Sep 05 13:42:41 picl0 systemd[1]: Starting Epilog for Flux job unknown...
Sep 05 13:42:41 picl0 epilog[1447]: Running epilog for unknown/unknown
Sep 05 13:42:41 picl0 epilog[1447]: running echo.sh
Sep 05 13:42:41 picl0 epilog[1450]: test-stdout
Sep 05 13:42:41 picl0 epilog[1450]: test-stderr
Sep 05 13:42:41 picl0 epilog[1447]: running sleep.sh
Sep 05 13:42:49 picl0 systemd[1]: flux-epilog@unknown.service: Succeeded.
Sep 05 13:42:49 picl0 systemd[1]: Finished Epilog for Flux job unknown.
```

Maybe a well placed comment in the script would be helpful?

Reply:
I didn't realize that. Comment is perfect.

Reply:
Thanks, squashed that and I'll set MWP.